openapi.yaml

```diff
@@ -226,6 +226,49 @@ paths:
               schema:
                 $ref: '#/components/schemas/Problem'
 
+  /optimize:
+    post:
```

Comment:
@magicmatatjahu if we provide the same arguments to the AsyncAPI optimizer, do we get the same result? I'm just thinking about caching in this endpoint since we can't have a `get` (we need to send a body payload), wdyt?

A `post` can't be cached..... but it can 🙂 (https://tech.ebayinc.com/engineering/caching-http-post-requests-and-responses/). If a user tries to optimize the same document multiple times, does the server have to do anything different between each run? If not this can be cached and we would improve response times.

would really like your feedback @magicmatatjahu and @smoya 👍